content/desktop/release-notes.md

```diff
@@ -24,11 +24,74 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/releases.md).
 
+# 4.24.0
+
+{{< release-date date="2023-09-28" >}}
+
+{{< desktop-install all=true version="4.24.0" build_path="/" >}}
+
+### New
```

Comment:
Think we need to mention Compose Watch is now GA in DD and link to https://docs.docker.com/compose/file-watch/

'Compose Watch is now generally available. For more information, see [Use Compose Watch](../compose/file-watch.md]'